server/jetstream_benchmark_test.go

```diff
@@ -1174,3 +1174,184 @@ func BenchmarkJetStreamKV(b *testing.B) {
 		)
 	}
 }
+
+func BenchmarkJetStreamObjStore(b *testing.B) {
+	const (
+		verbose            = false
+		objStoreNamePrefix = "B_"
+		keyPrefix          = "K_"
+		seed               = 12345
+		initKeys           = true
+
+		// read/write ratios
+		ReadOnly  = 1.0
+		WriteOnly = 0.0
+	)
+
+	// benchmark options matrix
+	benchmarkOptions := struct {
+		replicaAndClusterSize []int
+		numKeys               []int
+		objectSizeRange       [][]int
+		rwRatio               []float64
+		storageType           []nats.StorageType
+	}{
+		[]int{1, 3},            // 1 replica, 3 replicas
+		[]int{10, 1000, 10000}, // 10 keys, 1K keys, 10K keys
+		[][]int{
+			{1024, 1024},     // 1KB byte objects
+			{102400, 102400}, // 1MB byte objects
+			{1024, 102400},   // 1KB - 1MB byte objects
+		},
+		[]float64{ReadOnly, WriteOnly, 0.8},                      // readOnly, writeOnly, 80% read
+		[]nats.StorageType{nats.MemoryStorage, nats.FileStorage}, // memory retention, file retention
+	}
+
+	// run benchmarks using options matrix
+	for _, clusterSize := range benchmarkOptions.replicaAndClusterSize {
+		replicas := clusterSize
+		for _, numKeys := range benchmarkOptions.numKeys {
+			for _, objectSizeRange := range benchmarkOptions.objectSizeRange {
+				minObjSize, maxObjSize := objectSizeRange[0], objectSizeRange[1]
+				for _, rwRatio := range benchmarkOptions.rwRatio {
+					var rwRatioType string
+					switch rwRatio {
+					case 1.0:
+						rwRatioType = "readOnly"
+					case 0.0:
+						rwRatioType = "writeOnly"
+					default:
+						rwRatioType = fmt.Sprintf("rwRatio=%0.1f", rwRatio)
+					}
+					for _, storageType := range benchmarkOptions.storageType {
+						bName := fmt.Sprintf(
+							"N=%d,R=%d,K=%d,storageType=%s,minObjSize=%db,maxObjSize=%db,workload=%s",
+							clusterSize,
+							replicas,
+							numKeys,
+							storageType.String(),
+							minObjSize,
+							maxObjSize,
+							rwRatioType,
+						)
+						b.Run(
+							bName,
+							func(b *testing.B) {
+								rng := rand.New(rand.NewSource(int64(seed)))
+
+								if verbose {
+									b.Logf("Running %s workload %s with %d messages", bName, bName, b.N)
```

Comment:
`go bench` already prints bName for you. Also you are printing it twice in the same string.